DPG/Tasks/qaEventTrackLite.cxx

```diff
@@ -163,6 +163,14 @@ struct qaEventTrackLite {
   template <bool isMC, typename trackType>
   void fillHistograms(trackType const& track)
   {
+
+    if (TMath::Abs(track.dpgCollision().posZ() > 10.))
+      return;
+    if constexpr (isMC) {
+      if (track.productionMode() == 0) {
+        histos.fill(HIST("Particle/selPtEtaPhiMCGenPrimary"), track.ptMC(), track.etaMC(), track.phiMC());
```

Comment:
Would this not be filled twice? (line 286)

Reply:
This is the histo counting gen particles, it has to be filled both when looping over recoParticles and nonRecoparticles. Isn't that correct?

Reply:
Ah yes, you're right, apologies!
Ok approved for me!